iaso/api/form_versions.py

```diff
@@ -106,6 +106,8 @@ def validate(self, data: typing.MutableMapping):
             # keep xls_file empty to highlight the input in the UI
             raise serializers.ValidationError({"xls_file": "", "xls_file_validation_errors": validation_errors})
 
+        data["xls_file"].seek(0)
```

Comment:
Is there any reason why you're doing this here and not in `parse_xls_form()`?

Reply:
The validator "parse" the xls/xslx and the "parse_xls_rom" reparse it.
I think it only "breaks" when the file is a django "upload" file (like in the viewset)
and not when manipulating "plain" file (that's why I left the fix in the viewset, not sure a normal file have the seek(0))